static/js/main.js

```diff
@@ -0,0 +1,49 @@
+const toggleClass =  (el, cls) => {
+    if (Array.isArray(cls)) {
+        cls.map((cl) => {
+            el.classList.toggle(cl);
+        });
+    } else {
+        el.classList.toggle(cls);
+    }
+};
+
+function DarkModeIndex() {
+    var footer = document.getElementById('footer')
+    footer.classList.toggle("footer-dark")
+
+    toggleClass(document.getElementById('navbar'), ['bg-dark', 'navbar-dark']);
+    
+    toggleClass(document.getElementById('body'), ['body-dark']);
+    toggleClass(document.getElementById('darkmode'), ['body-dark']);
+    toggleClass(document.getElementById('title'), ['title-dark']);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/zlataovce/plshelp/pullRequest?prid=7177914)
```suggestion
    toggleClass(document.getElementById("title"), ['title-dark']);
```